### PR TITLE
BOTMETA.yml cleanup

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
BOTMETA.yml cleanup

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
`.github/BOTMETA.yml`

### CODE REVIEW

File: .github/BOTMETA.yml
Comment (gundalow) [inline .github/BOTMETA.yml:219]: A few lines above we have:
```yaml
  $modules/cloud/docker/:
    authors: chouseknecht
    maintainers: $team_docker
```

So you don't need to redefine `maintainers` for individual plugins unless you want to add extra people (which we  aren't)

File: .github/BOTMETA.yml
Comment (gundalow) [inline .github/BOTMETA.yml:297]: Please create a new entry above the `$modules/cloud/linode/linode.py:` entry for the directory and set the maintainers there.

File: .github/BOTMETA.yml
Comment (Andersson007) [reply .github/BOTMETA.yml]: done, thanks

File: .github/BOTMETA.yml
Comment (Andersson007) [reply .github/BOTMETA.yml]: done! i'll create an issue with suggestion adding authors to teams. If anybody agree, we can remove an extra visual noise and reduce BOTMETA size

File: .github/BOTMETA.yml
Comment (felixfontein) [inline .github/BOTMETA.yml:117]: I'm not sure whether this makes sense, since these seem to be distinct products.

File: .github/BOTMETA.yml
Comment (felixfontein) [inline .github/BOTMETA.yml:225]: Shouldn't `authors` entries be kept? (Except for dictories, like the above `authors: chouseknecht`.)

File: .github/BOTMETA.yml
Comment (felixfontein) [inline .github/BOTMETA.yml:423]: Why remove `DenBeke` here?

Comment (Andersson007) [reply .github/BOTMETA.yml:423]: Oh, thanks for noticing!

File: .github/BOTMETA.yml
Comment (Andersson007) [reply .github/BOTMETA.yml:117]: what do you mean? `team_cyberark_conjur` contains two other folks and is used in two places

File: .github/BOTMETA.yml
Comment (Andersson007) [reply .github/BOTMETA.yml:225]: the contributor is currently in `team_docker`
why should we keep authors when they are in teams that covers directories?

Comment (Andersson007) [reply .github/BOTMETA.yml:225]: @gundalow , i'm not sure you didn't mean the same yesterday, maybe we can get rid of `authors` completely?
The file is for internal usage by bot, isn't it? To notify people, count their shipits, etc., not for tracking authorship, right?
Is there any difference how the bot handles authors vs maintainers? If no difference, i think, one of them is extra
@gundalow @felixfontein what do you think?

Comment (felixfontein) [reply .github/BOTMETA.yml:225]: There's no difference for the bot, it's just semantics: if someone is later removed from the maintenance team for the group of modules, they are still authors of the module. If we removed them from `authors:` earlier, they are no longer listed there. (If they want to no longer receive notifications, IMO they should be added to `ignore:` and stay as `authors:`.)

Whether or not to keep `authors:` alltogether depends on what happens with the `authors:` in `DOCUMENTATION` validation.

Comment (Andersson007) [reply .github/BOTMETA.yml:225]: `If they want to no longer receive notifications, IMO they should be added to ignore: and stay as authors:`
can't we remove them from a team and put them to `maintainers:` of a certain module (where they are authors) in addition to $team_* instead?
e.g. `maintainers: usernicname $team_postgresql`

Comment (gundalow) [reply .github/BOTMETA.yml:225]: For people that are no longer involved, you are correct, they should be added to `ignore:`

Do we care about the difference between `author` and `maintainers`?
from a bot notification and merge (ship it) I believe they mean the same
We don't use BOTMETA when generating module docs, so nether maintainer nor author fields will be used to list authors on the webdocs

Comment (Andersson007) [reply .github/BOTMETA.yml:225]: cc @felixfontein

Comment (felixfontein) [reply .github/BOTMETA.yml:225]: It depends on what's the future of `authors:` in `DOCUMENTATION`. That's something we probably have to discuss in the community meeting at some point.

Comment (Andersson007) [reply .github/BOTMETA.yml:225]: Ok https://github.com/ansible/community/issues/539#issuecomment-706947663

**Comment by Andersson007** [comment]
@gundalow @felixfontein could we merge this as it doesn't remove `authors` completely? I wouldn't like this will be hanging here for indefinite period of time. I'd start the issue and PR to update / expand the teams after this is merged.

**Comment by felixfontein** [comment]
@Andersson007 I'd really prefer if you could keep the `authors:` fields for now.

**Comment by Andersson007** [comment]
> @Andersson007 I'd really prefer if you could keep the `authors:` fields for now.

@felixfontein i put them back (didn't do it for directory level). PTAL

File: .github/BOTMETA.yml
Comment (felixfontein) [reply .github/BOTMETA.yml:117]: [Conjur](https://www.conjur.org/) (which is open-source) is another product from Cyberark than [Password](https://www.cyberark.com/products/privileged-account-security-solution/enterprise-password-vault/) (which is closed-source). I don't see why the conjur maintainers should have anything to do with the cyberarkpassword lookup, since it is for a different product.

**Review by felixfontein — COMMENTED** [review]
Looks good, except possibly `team_cyberark_conjur` for `$lookups/cyberarkpassword.py` (see above).

File: .github/BOTMETA.yml
Comment (Andersson007) [reply .github/BOTMETA.yml:116]: removed

**Comment by Andersson007** [comment]
@felixfontein PTAL

**Review by felixfontein — APPROVED** [review]
LGTM

**Comment by Andersson007** [comment]
@gundalow do you have anything to add? if no, feel free to merge:)

File: .github/BOTMETA.yml
Comment (Andersson007) [reply .github/BOTMETA.yml]: fixed, thanks
xenserver stuff should also be updated, i'll do it

**Review by gundalow — APPROVED** [review]

**Comment by gundalow** [comment]
@Andersson007 @felixfontein Thank you for this

**Pull request merged**
